static/authui/js/main.js

```diff
@@ -158,9 +160,14 @@ window.addEventListener("load", function() {
         displaySeconds = Math.round((cooldown - timeElapsed) / 1000);
       }
 
-      el.textContent = (el.textContent || "").replace(/\d+/, String(displaySeconds));
+      if (displaySeconds === 0) {
+        el.textContent = label;
+      } else {
+        el.textContent = labelUnit.replace("%d", String(displaySeconds));
+      }
     }
-    token = setInterval(tick, 500);
+    // 24fps => 1 / 24 * 1000 = 42ms
+    token = setInterval(tick, 42);
```

Comment:
Actually, why do we need to use smaller interval?

Reply:
0.5s sometimes does not look smooth.

Reply:
Let's use `requestAnimationFrame` instead?

Reply:
`requestAnimationFrame` is a bit over kill.

Reply:
I don't think it's overkill. The issue we have at hand is 'countdown animation is not smooth', so using a proper animation API to update it is quite appropriate.